client/blocks/location-search/style.scss

```diff
@@ -1,7 +1,7 @@
 .location-search__prediction {
 	cursor: pointer;
 	&:hover {
-		color: lighten( $blue-wordpress, 5% );
+		color: var( --color-primary-300 );
```

Comment:
I think this could be `--color-primary-light`. Not necessary, but just a suggestion.